content/dotnet-template-azure-iot-edge-module/CSharp/Dockerfile.amd64.debug

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.0-runtime-stretch AS base
+FROM microsoft/dotnet:2.1-runtime-stretch-slim AS base

```

Comment:
There is inconsistency between amd64 and amd64.debug: microsoft/dotnet:2.1-runtime-alpine vs. microsoft/dotnet:2.1-runtime-stretch-slim

Reply:
Yes, alpine image is very small. However it cannot work well with vsdbg currently.